src/site/markdown/server/manage-webadmin.md

```diff
@@ -533,6 +532,20 @@ The scheduled task will have the following type `RecomputeAllPreviewsTask` and t
 }
 ```
 
+An admin can specify the concurrency that should be used when running the task:
```

Comment:
we may illustrate how these parameter combines:
userParallelism = 2 / mailboxParallelism = 4 / messageParallelism = 1 means 8 concurrent messages indexed at the same time.